action.yml

```diff
@@ -72,5 +75,12 @@ runs:
           echo "semver-type=minor" >> $GITHUB_OUTPUT
           exit 0
         fi
+        EOF=$(dd if=/dev/urandom bs=15 count=1 status=none | base64)
         echo "semver-type=major" >> $GITHUB_OUTPUT
+        {
+          echo "output<<$EOF"
+          echo "$OUTPUT"
+          echo "$EOF"
+        } >> $GITHUB_OUTPUT
```

Comment:
Seems like we'll only be setting the `output` variable if `semver-type=major`. I think this needs to move up above the semver-type checks.